components/specification/publish

```diff
@@ -437,40 +438,18 @@ createintermediateindex() {
 <h2>Table of contents</h2>
         <ol>
                 <li><a href="#intro">Introduction</a></li>
-                <li><a href="#history">History</a></li>
                 <li><a href="#schema.list">Schemas</a></li>
+                <li><a href="#history">History</a></li>
         </ol>
 </div>
 <div id="intro">
 <h2>Introduction</h2>
-<p>This document outlines the XML $1 Schemas created by the Open Microscopy Environment Group. This is the $(schemadesc "$1")  Detailed information on the schemas and future development plans are available at <a href="http://www.openmicroscopy.org/site/support/file-formats">http://www.openmicroscopy.org/site/support/file-formats</a>. Further information on the groups work is available at <a href="http://www.openmicroscopy.org/">http://www.openmicroscopy.org/</a>
+<p>This document outlines the XML $1 Schemas created by the Open Microscopy Environment Group. This is the $(schemadesc "$1")  Detailed information on the schemas and future development plans are available at <a href="http://www.openmicroscopy.org/site/support/ome-model/index.html">http://www.openmicroscopy.org/site/support/ome-model/index.html</a>. The main OME website is available at <a href="http://www.openmicroscopy.org/">http://www.openmicroscopy.org/</a>
```

Comment:
Same as above, I think we should remove the reference to future development plans